template-xml/app/src/test/java/co/nimblehq/template/xml/ui/screens/xml/HomeFragmentTest.kt

```diff
@@ -6,6 +6,8 @@ import co.nimblehq.template.xml.test.TestNavigatorModule.mockMainNavigator
 import co.nimblehq.template.xml.test.getPrivateProperty
 import co.nimblehq.template.xml.test.replace
 import co.nimblehq.template.xml.ui.BaseFragmentTest
+import co.nimblehq.template.xml.ui.screens.home.HomeFragment
+import co.nimblehq.template.xml.ui.screens.home.HomeViewModel
```

Comment:
Please move this `HomeFragmentTest` file to `package co.nimblehq.template.xml.ui.screens.home`

<img width="349" alt="image" src="https://user-images.githubusercontent.com/16315358/203493868-3f1cb613-13cd-4cb0-aa47-fd16042debac.png">